packages/wordpress-playground-block/src/components/playground-preview/index.tsx

```diff
@@ -594,11 +641,22 @@ export default function PlaygroundPreview({
 					className="demo-footer__link"
 					target="_blank"
 				>
-					<span className="demo-footer__powered">Powered by</span>
-					<Icon className="demo-footer__icon" icon={wordpress} />
-					<span className="demo-footer__link-text">
-						WordPress Playground
-					</span>
+					{createInterpolateElement(
+						// translators: powered-by label with embedded icon. please leave markup tags intact, including numbering.
+						__(
+							'<span1>Powered by</span1> <Icon /> <span2>WordPress Playground</span2>'
```

Comment:
The signature of `createInterpolateElement()` takes a map with keys for element names, so it is not possible to use regular `<span>` twice here and have them treated separately. But it turns out we can use unique non-existent tag names and achieve what we want.

It's not great because it might confuse translators who might "fix" our span "typos". To attempt to reduce the risk of this, I added a translators note that mentions the numbered markup tags.